modules/scheduler/README.md

```diff
@@ -0,0 +1,65 @@
+# Instance Scheduler
+This module creates Lambda functions that turn on or off EC2 Instances, Auto Scaling Groups, RDS Instances, ECS Clusters, RedShift Clusters, and DocumentDB Instances based on the instance tags. The code is sourced from the [diodonfrost terraform-aws-lambda-scheduler-stop-start github](https://github.com/diodonfrost/terraform-aws-lambda-scheduler-stop-start), with parameters entered to configure the function, including a cron expression for the schedule.
```

Comment:
This gives us a dependency on the mentioned github repo so we will need to keep an eye on that in the future. Just a headsup.

Reply:
Yep, it typically only gets an update when the Python version needs changing which the current version being used got updated to Python 3.10 last year